solr/solr-ref-guide/modules/configuration-guide/pages/caches-warming.adoc

```diff
@@ -235,6 +235,18 @@ This is the same system property used in the xref:configuring-solr-xml.adoc#glob
 <maxBooleanClauses>${solr.max.booleanClauses:1024}</maxBooleanClauses>
 ----
 
+=== <minPrefixLength> Element
```

Comment:
hmm; weird that a documentation file named "caches-warming" would document this query specific limit

Reply:
Agreed.  Looking at the file it, it covers everything in the `<query>` section of solrconfig.xml.  A bunch of that _is_ cache related, but not all of it obviously.

Maybe the scope of the file changed over time and no one ever noticed as the name became more and more out of date?  Or perhaps someone noticed, but the name was intentionally retained for SEO or other reasons since it appears in the ref-guide URL?

Reply:
I don't think we should keep this named as such for legacy SEO purposes.  Any way; it'd be a follow-up PR, not here.

Reply:
Yeah, maybe not SEO purposes.  But I know there have been discussions about tracking changes to page names (I think for the purposes of the version-switcher our ref-guide has now?)

Anyway, something for another day...